modules/ROOT/pages/workday/workday-connector-design-center.adoc

```diff
@@ -1,63 +1,67 @@
-= Workday Design Center Configuration - Mule 4
+= Workday - Design Center Configuration - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
-
 :page-aliases: workday-design-center.adoc
 
-The Flow Designer feature of Design Center enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a Design environment. For more information, see the Flow Designer Tour.
+Anypoint Design Center's Flow Designer enables you to create apps visually. To use Flow Designer, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
 
```

Comment:
I still think we can use "design environment" as a general type, rather than a name.... But you own it!  :-)